Lombiq.HelpfulLibraries.LinqToDb/Docs/LinqToDb.md

```diff
@@ -0,0 +1,45 @@
+# Lombiq Linq2DB for Orchard Core

```

Comment:
Inkább egy Readme fájl legyen a project rootban, és a RestEase-essel is ezt csináld meg légyszi.